client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -23,13 +34,92 @@ const isMonthsOld = ( months: number, rawDate?: string ) => {
 	return moment().diff( parsedDate, 'months' ) > months;
 };
 
-export default function PlanOnlyThankYou( { primaryPurchase }: PlanOnlyThankYouProps ) {
+export default function PlanOnlyThankYou( {
+	primaryPurchase,
+	isEmailVerified,
+}: PlanOnlyThankYouProps ) {
 	const siteId = useSelector( getSelectedSiteId );
 	const siteSlug = useSelector( getSelectedSiteSlug );
+	const emailAddress = useSelector( getCurrentUserEmail );
 	const siteCreatedTimeStamp = useSelector(
 		( state ) => getSiteOptions( state, siteId ?? 0 )?.created_at
 	);
 
+	const [ resendStatus, setResendStatus ] = useState( RESEND_NOT_SENT );
+
+	const resendEmail = () => {
+		removeNotice( VERIFY_EMAIL_ERROR_NOTICE );
+
+		if ( RESEND_PENDING === resendStatus ) {
+			return;
+		}
+
+		setResendStatus( RESEND_PENDING );
+
+		wpcom.req.post( '/me/send-verification-email', ( error: Error ) => {
+			if ( error ) {
+				errorNotice( translate( "Couldn't resend verification email. Please try again." ), {
+					id: VERIFY_EMAIL_ERROR_NOTICE,
+				} );
+
+				setResendStatus( RESEND_ERROR );
+				return;
+			}
+
+			setResendStatus( RESEND_SUCCESS );
+		} );
+	};
+
+	const resendButtonText = () => {
+		switch ( resendStatus ) {
+			case RESEND_PENDING:
+				return translate( 'Sending…' );
+			case RESEND_SUCCESS:
+				return translate( 'Email sent' );
+			case RESEND_NOT_SENT:
+			case RESEND_ERROR:
+			default:
+				return translate( 'Resend email' );
+		}
+	};
+
+	let subtitle;
+	let headerButtons;
+	if ( primaryPurchase.productSlug === 'ecommerce-bundle' ) {
+		if ( isEmailVerified ) {
+			subtitle = translate( "With the plan sorted, it's time to start setting up your store." );
+			headerButtons = (
+				<Button
+					variant="primary"
+					href={ `http://${ siteSlug }/wp-admin/admin.php?page=wc-admin&from-calypso` }
+				>
+					Create your store
+				</Button>
+			);
+		} else {
+			subtitle = translate(
+				"{{paragraph}}With the plan sorted, verify your email address to create your store.{{br/}}Please click the link in the email we sent to %(emailAddress)s.{{/paragraph}}{{paragraph}}If you haven't received the verification email, please click here.{{/paragraph}}",
+				{
+					args: { emailAddress: emailAddress },
+					components: { paragraph: <p />, br: <br /> },
+				}
+			);
+			headerButtons = (
+				<Button variant="secondary" onClick={ resendEmail }>
+					{ resendButtonText() }
+				</Button>
+			);
```

Comment:
I'd like to suggest a few UX improvements here:
* Bold the email address so that it's more prominent.
* Apply a "busy" state to the button when the email is being sent (this ensures we remain consistent with buttons in other Congrats pages. See: https://github.com/Automattic/wp-calypso/blob/02c2989e213b75fc9bd24bc6db579ff720ea489d/client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you-theme-section.tsx#L122) Note that we'll need to adjust the import from `@wordpress/components` to `@automattic/components` for the `<Button>` component in order take advantage of the visual treatment for the busy state.
* Make "click here" a link and call `resendEmail` when clicked. It seemed weird that we're showing the text "click here" when it isn't clickable.

https://github.com/Automattic/wp-calypso/assets/104910361/93c78c95-8224-44a6-8f41-cd28203ee92b

```suggestion
			subtitle = translate(
				"{{paragraph}}With the plan sorted, verify your email address to create your store.{{br/}}Please click the link in the email we sent to {{strong}}%(emailAddress)s{{/strong}}.{{/paragraph}}{{paragraph}}If you haven't received the verification email, please {{a}}click here{{/a}}.{{/paragraph}}",
				{
					args: { emailAddress: emailAddress },
					components: {
					    paragraph: <p />,
					    br: <br />,
					    strong: <strong />,
					    a: <a href="" onClick={ resendEmail },
					},
				}
			);
			const isSendingEmail = resendStatus === RESEND_PENDING;
			headerButtons = (
				<Button variant="secondary" onClick={ resendEmail } busy={ isSendingEmail } disabled={ isSendingEmail }>
					{ resendButtonText() }
				</Button>
			);
```

Reply:
This is excellent, thank you.

Applying the diff didn't quite work, as the linter threw an a11y warning on the anchor. Once it has an `onClick` it points out that we're using an anchor for a button when those are intended for navigation. I've set an ignore on the line for now. Would you prefer we ignore that lint rule in this case? Or use an actual `Button` with a combination of props and CSS to make it look right?